addons/jest/src/components/Panel.tsx

```diff
@@ -68,37 +71,25 @@ const SuiteProgress = styled(({ successNumber, result, className }) => (
     <span style={{ width: `${(successNumber / result.assertionResults.length) * 100}%` }} />
   </div>
 ))(() => ({
-  width: '100%',
-  backgroundColor: colors.error,
-  height: 4,
+  width: '50px',
+  backgroundColor: '#FF4400',
+  height: '10px',
   top: 0,
   position: 'absolute',
   left: 0,
-  borderRadius: 3,
   overflow: 'hidden',
   appearance: 'none',
 
   '& > span': {
-    backgroundColor: colors.success,
+    backgroundColor: '#66BF3C',
     bottom: 0,
     position: 'absolute',
     left: 0,
     top: 0,
-    boxShadow: '4px 0 0 white',
+    boxShadow: '1px 0 0 white',
```

Comment:
From a visual acuity standpoint adding 1px white border between the red and green doesn't add much. Can we remove this?